de/portal.json

```diff
@@ -71,18 +71,18 @@
             "is online now": "ist gerade online!",
             "is offline now": "ist offline",
             "deviceName is powerState now": "$deviceName ist jetzt $powerState.",
-            "Please verify email address": "Bitte überprüfen Sie Ihre E-Mail-Adresse, um auf alle Funktionen von Sinric zugreifen zu können",
+            "Please verify email address": "Bitte überprüfen Sie Ihre E-Mail Adresse, um auf alle Funktionen von Sinric zugreifen zu können",
             "devices": "Geräte",
             "Ding Dong": "Ding Dong!",
             "Power is": "Macht ist",
```

Comment:
What is meant with this? It makes no sense in German.

Reply:
"Bitte bestätigen Sie Ihre E-Mail Adresse" ist damit gemeint.